muesli/utils.py

```diff
@@ -53,6 +53,14 @@
         {'id': 'presence_assignment', 'name': 'Präsenzübung'},
         {'id': 'mock_exam', 'name': 'Probeklausur'}]
 
+category_names = {
```

Comment:
Warum wird hier denn das redefiniert was drüber schon steht? :thinking:

Reply:
Fixed in the newest commit